src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -94,46 +80,84 @@ public String saveVolunteer(Volunteer volunteer, Model model){
 
     /**
      * PostMapping for /volunteer endpoint
-     *Create a nwe volunteer with details provided via JSON
+     * Create a new volunteer with details provided via JSON
      *
      * Returns one of the following status codes:
-     * 201:
-     * 409:
+     * 201: Created
+     * 409: Conflict - Volunteer likely already exists
```

Comment:
Added detail to comments where doable to reduce lack of comments, a SonarQube Deadly Sin. [Reference](https://docs.sonarqube.org/display/SONARQUBE45/Not+Enough+or+Too+Many+Comments)

Reply:
Nice definition for status codes.